Src/BootCamp.Chapter/EventManager.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class EventManager
+    {
+        public static Events ManageEvents()
+        {
+            var events = new Events();
+            events.DemoStarted += OnDemoStarted;
+            events.DemoEnded += OnDemoEnded;
+            events.ChoosePredicate += ChoosePredicates;
+            events.ApplicationClosed += OnApplicationClosed;
+
+            return events; 
+        }
+
+        protected static void OnDemoStarted(object sender, EventArgs e)
+        {
+            Console.WriteLine("The demo has started");
+        }
+
+        protected static void OnDemoEnded(object sender, EventArgs e)
+        {
+            Console.WriteLine("the demo has ended");
+        }
+
+        protected static void OnApplicationClosed(object sender, EventArgs e)
+        {
+            Console.WriteLine("the application has closed");
+        }
+
+        protected static void ChoosePredicates(object sender, EventArgs e)
+        {
+            var menuChoice = MenuPredicates();
+            var contact = new ContactsCenter(@"Input/MOCK_DATA.csv");
+            var persons = new List<Person>();
+
+            var answer = new List<Person>();
+            switch (menuChoice)
+            {
+                case "1":
+                    answer = contact.Filter(PeoplePredicates.IsA);
```

Comment:
This is not an answer, answer is actually "1". This is a filter.

Reply:
oke, I will renamed answer to filter. I named this answer because I thought that it holds all the persons which are filtered out by the predicate.  But also maybe I misunderstood things

Reply:
no, answer is a list mqaybe I can better name it filteredData or filteredPeople.
see : 
![afbeelding](https://user-images.githubusercontent.com/59806635/78425058-37a30000-7672-11ea-9739-265c2080be95.png)